src/Intervention/Image/ImageManagerStatic.php

```diff
@@ -20,7 +20,7 @@ class ImageManagerStatic
      */
     public function __construct(ImageManager $manager = null)
     {
-        self::$manager = $manager ? $manager : new ImageManager;
+        self::$manager = $manager ? $manager : resolve(ImageManager::class);
```

Comment:
You can't do this because the library is not just for Laravel, can't call a framework-specific global helper.

Reply:
ok, i'm fixed it